PokemonWorkshop/PokemonStudio
# Fix #21 + upgrade electron for new html features

## Description

Fix #21
This PR simply adds a new option for the move target: All ally but user. (IDK why we have to use such a long sentence with the meaning of the sentence requiring hovering the option, we should improve that with tooltip).

## Note before testing

This PR is connected to this MR: https://gitlab.com/pokemonsdk/pokemonsdk/-/merge_requests/1164
To test in PSDK you need to run this PR, edit the move Coaching so it use this new target type, save the project and use the move in PSDK with the MR loaded.

## Tests to perform

1. Load a project, edit a move with this new target type, save.
2. Close studio, load this project again (it should load properly).
3. Test the PSDK MR according to above notes.
